tools/make-directive-index.py

```diff
@@ -160,6 +160,38 @@
                 <variablelist id='filenames' />
         </refsect1>
 
+        <refsect1>
+                <title>d-bus interfaces</title>
+
+                <para>Interaces exposed on the D-BUS.</para>
```

Comment:
"Interfaces exposed over D-Bus."
(D-Bus is a name, hence "the" should not be used.)